packages/desktop/components/popup/popups/SyncAccountsPopup.svelte

```diff
@@ -60,8 +55,69 @@
         }
     }
 
+    const networkSearchMethod: { [key in StardustNetworkId]?: () => Promise<void> } = {
+        [StardustNetworkId.Iota]: multiAddressSearch,
+        [StardustNetworkId.Shimmer]: singleAddressSearch,
+        [StardustNetworkId.Testnet]: singleAddressSearch,
+    }
+
+    async function singleAddressSearch(): Promise<void> {
+        const recoverAccountsPayload: RecoverAccountsPayload = {
+            accountStartIndex,
+            accountGapLimit,
+            addressGapLimit: 1,
+            syncOptions: { ...DEFAULT_SYNC_OPTIONS, addressStartIndex: 0 },
+        }
+
+        await recoverAccounts(recoverAccountsPayload)
+
+        accountStartIndex++
```

Comment:
we are just incrementing the account start index by 1 instead of the expected range?